src/request/sign-swap/SignSwap.js

```diff
@@ -164,7 +168,8 @@ class SignSwap {
         const exchangeRateString = `1 ${exchangeBaseAsset} = ${NumberFormatting.formatNumber(
             exchangeRate,
             exchangeRateDecimals,
-            exchangeOtherAsset === 'EUR' ? CryptoUtils.assetDecimals(exchangeOtherAsset) : 0,
+            exchangeOtherAsset === 'EUR' || exchangeOtherAsset === 'CRC'
+                ? CryptoUtils.assetDecimals(exchangeOtherAsset) : 0,
```

Comment:
Here too, I wouldn't enforce `CryptoUtils.assetDecimals(exchangeOtherAsset)` (-> 2) minimum decimal places for CRC.